rqpy/io/_load.py

```diff
@@ -237,7 +237,7 @@ def getrandevents(basepath, evtnums, seriesnums, cut=None, channels=["PDS1"], de
     return t, x, crand
 
 
-def get_trace_gain(path, chan, det, gainfactors={'rfb': 5000, 'loopgain' : 2.4, 'adcpervolt' : 2**(16)/2, 'lowpassgain' : 2}):
+def get_trace_gain(path, chan, det, rfb=5000, loopgain=2.4, adcpervolt=2**(16)/8, lowpassgain=4):
```

Comment:
are the values for adcpervolt and lowpassgain stored somewhere that is tied to the data somehow? As in, can the user access this data without knowing what dcrc was used? If so, it would be nice to just get rid of these

Reply:
The adcpervolt and lowpassgain are not stored in the data as fas as i know. However we can get the DCRC version in the raw data so we could automatically assigned the appropriate values according to DCRC version and keep arguments as None in case gain is modified for R&D purpose  (as done in the past).

Reply:
Yes, I like that idea, assign the values based on what DCRC version was used to take the data, but leave them as optional parameters in case the user wants to change them

Reply:
I can try to  implement today though I didn't change from what it is in master now so this also a  bit beyond the work on this branch....

Reply:
I don't think it's absolutely necessary for this PR, we could make an issue to remind us to do this?

Reply:
Yeah, we can just open a separate issue for this

Reply:
Cool, I'll make one now.

Reply:
Created, #167. I'll resolve this conversation.

Reply:
Just as a general comment, many more changes of these functions will need to be done to analyze data from the new cyroconcept fridge. The "adcpervolt" coefficient for example will be  part of the raw data  as it is read from national instrument ADC card so we will need to make the arguments optional in any cases. The most important update  is starting after this branch is merged.... :-)

Reply:
Actually, I just realized that if you don't pass the trace gain to `get_traces_midgz()`, it will call `get_trace_gain()` with the default values. https://github.com/ucbpylegroup/RQpy/blob/feature/rawio/rqpy/io/_load.py#L362 which will clearly be wrong depending on what DCRC was used. So we either need to do this change first, or remove that functionality from `get_traces_midgz()`

Reply:
Perhaps we should add this to the #167 issue? I agree with @serfass that this doesn't really fall within the scope of this PR.